app/services/client_api/invitations_service.rb

```diff
@@ -0,0 +1,161 @@
+module ClientApi
+  class InvitationsService
+    include InputSanitizeHelper
+    include UsersGenerator
+
+    def initialize(args)
+      @user = args[:user]
+      @emails = args[:emails].map(&:downcase)
+      @team = args[:team]
+      @role = args[:role]
+
+      raise ClientApi::CustomInvitationsError unless @emails && @team && @role
+      @emails && @emails.empty? { raise ClientApi::CustomInvitationsError }
```

Comment:
why double checking?

Reply:
@emails could exist, but the array could be empty.

Reply:
I meant on line 12 and line 13